### ISSUE TITLE
Dashboard crashes on 1.6.0 Beta1 and Beta2 on iOS

### ISSUE THREAD

**Issue by lexusburn** [Clooos/Bubble-Card#256]
https://github.com/Clooos/Bubble-Card/assets/659847/38c16dea-c30c-40f9-8c77-208f6b4318a5

I still have the crashes on iOS when using version 1.6. It's the same behavior in iOS App and iOS Safari. 

No JavaScript Errors in console. 

On desktop it's working fine. 

After downgrade it's working again on iOS. 

Had tried to clear caches and using timestamp on ressources to get latest version of JavaScript libraries.

**Comment by Clooos** [comment]
Hi, do you have any logs that might be related to these crashes? 

Settings > System > Logs

**Comment by lexusburn** [comment]
No, nothing related to this crashes. :-(

Is there any possibility to raise loglevel for the dashboard?

Downgrad to 1.5.3 is working. 
Both new beta versions are not working at all with iOS.

I'm using the the optimized loading for your bubble-card extension and it's an yaml-created dashboard which is using lovelace_gen to be generated based on some macros (templates).

Here the overview from javascript console (no errors):
```
[Info]  Bubble Card - Pop-up  v1.6.0-beta.2  (bubble-pop-up.js, line 298)
[Info] CARD-MOD 3.3.0 IS INSTALLED (card-mod.js, line 5)
[Info]   ALARMO-CARD   (alarmo-card.js, line 784)  Version: v1.5.1 
[Info]  APEXCHARTS-CARD  v2.0.4  (apexcharts-card.js, line 835)
[Info]   BAR-CARD  (bar-card.js, line 4855)  Version 3.1.7    
[Info]   BUTTON-CARD   (button-card.js, line 426) Version 4.1.1 
[Log] AUTO-ENTITIES 1.12.1 IS INSTALLED (auto-entities.js, line 172)
[Log] Readme: – "https://github.com/thomasloven/lovelace-auto-entities" (auto-entities.js, line 172)
[Info] BADGE-CARD 1.0.1 IS INSTALLED – "" (badge-card.js, line 6)
[Info]   HOURLY-WEATHER-CARD  (hourly-weather.js, line 304)  Version 5.3.0    
[Info] 🍄 Mushroom 🍄 - 3.2.2 (mushroom.js, line 3557)
[Info] STATE-SWITCH 1.9.5 IS INSTALLED – "" (state-switch.js, line 139)
[Info]  MINI-GRAPH-CARD  0.11.0  (mini-graph-card-bundle.js, line 1)
[Info]   ---- MY-SLIDER ----  (my-cards.js, line 186)  Version 3.0.4    
[Info]   ---- MY-SLIDER-V2 ----  (my-cards.js, line 356)  Version 3.0.4    
[Info]   ---- MY-BUTTON ----  (my-cards.js, line 375)  Version 1.0.0    
[Info]   ---- MY-CARDS ----  (my-cards.js, line 436)  Version 2.0.2    
[Info]   SCHEDULER-CARD   (scheduler-card.js, line 1826)   Version: v3.2.12
[Info]  TIMER-BAR-CARD  Version 1.29.1  (timer-bar-card.js, line 474)
[Info]  Bubble Card  v1.6.0-beta.2  (bubble-card.js, line 485)
[Info]  FRIGATE-HASS-CARD  (card-555679fd.js, line 507) Version 5.2.0 
```

**Comment by Clooos** [comment]
I've added this in my `configuration.yaml` but I don't remember if it change something:
```yaml
logger:
  default: warning
```

Without any error I'm clueless right now 😅

**Comment by lexusburn** [comment]
> ```yaml
> logger:
>   default: warning
> ```

Tried this, but i can't get any logs regarding the dashboard rendering.
It seems that the only logs regarding this would be in browser javascript console.

But i got good news:
I restarted the whole iOS and it had only a dashboard reload after the initial start and it's running fine since some minutes now. 
A dashboard reload by hand is working too and i've didn't get the crashing behavior you see in the video.
Hopefully the problem with the crashing dashboard does not occur anymore.

**Therefore my suggestion for all other users with a problem:**

- **clear caches**
- **restart your device**
